bin/programs/client/src/kona.rs

```diff
@@ -21,7 +21,7 @@ extern crate alloc;
 /// The size of the LRU cache in the oracle.
 const ORACLE_LRU_SIZE: usize = 1024;
 
-#[client_entry(0x77359400)]
+#[client_entry(100_000_000)]
```

Comment:
Note - we're gonna have to play with this, but 100MB is far more than we need on avg.